wrapanapi/systems/openstack.py

```diff
@@ -27,6 +27,7 @@
 from novaclient import exceptions as os_exceptions
 from novaclient.client import SessionClient
 from novaclient.v2.floating_ips import FloatingIP
+from openstack.connection import Connection
```

Comment:
We don't directly require openstack SDK, we're only getting it through our dependency on python-ironicclient.

You need to update the requirements file to specify a version of openstacksdk.

Running this with current requirements, we're getting openstacksdk==0.27.0 installed, and the latest is 0.32.0.

```[WARNING] [py.warnings] ./.cfme_venv/lib64/python3.7/site-packages/openstack/_meta/connection.py:121: ImportWarning: Could not import data-processing service description: No module named 'openstack.data_processing'
  ImportWarning)
```

This version, along with our other package versions, is resulting in the above import error - along with a few dozen others from connection module.

Please evaluate the package version needs for python-*client and python-openstacksdk or openstacksdk requirement.

Reply:
@ldjebran I'm willing to accept the OpenstackSystem.create_vm method, as its part of the original design  as an abstract method.

The above needs to be resolved though for the PR to be accepted